src/Login/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Input,
+  FormControl,
+  Heading,
+  VStack,
+  useToast,
+} from 'native-base';
+import Constants from 'expo-constants';
+
+const API_URL = Constants?.manifest?.extra?.API_URL;
+
+export const LoginScreen = (props: LoginProps) => {
+  const [show, setShow] = React.useState<boolean>(false);
+  const [username, setUsername] = React.useState<string>('');
+  const [password, setPassword] = React.useState<string>('');
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [errorMessage, setErrorMessage] = React.useState<errorMessage>();
+
+  const toast = useToast();
+
+  const handlePasswordShow = () => setShow(!show);
+
+  const handleLogin = async (usernameValue: string, passwordValue: string) => {
```

Comment:
You don't need to pass the `username` and `password` as parameters, they are already in the scope.